File: public/docs/_examples/security/ts/app/inner-html-binding.component.ts
// #docregion
import { Component } from '@angular/core';

@Component({
  moduleId: module.id,
  selector: 'inner-html-binding',
  templateUrl: 'inner-html-binding.component.html',
})
// #docregion inner-html-controller
export class InnerHtmlBindingComponent {
  // E.g. a user/attacker controlled value from a URL.
  htmlSnippet = 'Template <script>alert("0wned")</script> <b>Syntax</b>';
}

File: public/docs/_examples/security/dart/lib/inner_html_binding_component.dart
// #docregion
import 'package:angular2/core.dart';

@Component(
    selector: 'inner-html-binding',
    templateUrl: 'inner_html_binding_component.html')
// #docregion class
class InnerHtmlBindingComponent {
  // E.g. a user/attacker controlled value from a URL.
  var htmlSnippet = 'Template <script>alert("0wned")</script> <b>Syntax</b>';
}
